src/sidebar/CustomModelBox.tsx

```diff
@@ -129,6 +131,11 @@ export default function CustomModelBox({ enabled, encodedValues, initialCustomMo
         [editor, isValid]
     )
 
+    const isQueryOngoing = () => {
+        if (!getQueryStore().state.currentRequest.subRequests[0]) return false
+        return getQueryStore().state.currentRequest.subRequests[0].state == RequestState.SENT
```

Comment:
Is this the same as `getQueryStore().state.currentRequest.subRequests[0]?.state === RequestState.SENT`?

Reply:
... and not sure if it is a good idea to access the state like this, maybe we can pass this boolean into CustomModelBox as a prop? I don't know, just wondering.

Reply:
I think it would be the first time we do it differently, so better not. I changed this here and it seems to work: 686e97919fe03882e0423ee7650bf8539fe7537c